sdk/containerregistry/Azure.Containers.ContainerRegistry/samples/Sample02a_DeleteImages.md

```diff
@@ -0,0 +1,34 @@
+# Azure.Containers.ContainerRegistry Samples - Delete Images (sync)
+
+A common use case for Azure Container Registries is to scan the repositories in a registry and delete all the images older than a certain date, or to keep the latest n images.  This sample illustrates how to use the .NET ACR SDK to delete all but the latest three images.
```

Comment:
This wording threw me off a bit. Is there a functional difference between deleting images older than x and keeping the latest images?

Reply:
Yes - you could say - delete all images that were lastUpdated before June 1.  Or, you could say - delete all images except the three newest.  In a lot of cases, this would have a different result.

Reply:
Makes sense. Perhaps it would be more clear and general to say that one could delete entities relative to the value of the ordered property or their position in the ordered result